app/src/main/java/com/example/xpvehicles/activities/MainActivity.java

```diff
@@ -116,6 +116,7 @@ public void onLocationChanged(@NonNull Location location) {
                 userLocationGeoPoint = new ParseGeoPoint(userLatitude, userLongitude);
                 Log.i(TAG, "User's location is " + userLocationGeoPoint);
                 exploreFragment.notifyAdapter();
+                savedFragment.notifyAdapter();
```

Comment:
why do we need this?

Reply:
In case the user clicks on the saved fragment before the location has been determined, the saved fragment still need to update the adapter when the location has been determined for the distances can be displayed

Reply:
I thought we have loading spinner now?

Reply:
The loading spinner is added to each fragment xml file, so I still need to notify the adapter to set the visibility of the spinner to gone.